### ISSUE TITLE
GUI geht nicht mehr auf mit 2.2.0

### ISSUE THREAD

**Issue by Little-Ben** [tim4724/u-song-Stage-Monitor-for-Songbeamer#7]
Leider geht keines der usong Fenster mehr auf mit der neuen Version.
Hatte vorher die regkeys komplett gelöscht.
Die regKeys für UI bleiben dann leer (bzw nicht vorhanden).
Ein Starten der alten Version (2.1.0) legt die keys dann wieder an, allerdings bringt das auch keine Besserung - allerdings ein "Settings2".

Habe 2 1080p Bildschirme

**Comment by Little-Ben** [comment]
regkeys komplett gelöscht, 2.2.0 neugestartet, regkeys export:

Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Software\JavaSoft\Prefs\com\tim]

[HKEY_CURRENT_USER\Software\JavaSoft\Prefs\com\tim\usong]
"first_run"="false"

[HKEY_CURRENT_USER\Software\JavaSoft\Prefs\com\tim\usong\preferences]
"song/Dir"="/D:///Data///Documents///Song/Beamer///Songs"
"title/Has/Page"="true"
"max/Lines/Page"="3"
"chords"="false"
"chords/Use/B/Natural"="false"
"show/Preview"="true"

[HKEY_CURRENT_USER\Software\JavaSoft\Prefs\com\tim\usong\ui]

[HKEY_CURRENT_USER\Software\JavaSoft\Prefs\com\tim\usong\ui\preview]

[HKEY_CURRENT_USER\Software\JavaSoft\Prefs\com\tim\usong\ui\tutorial]

[HKEY_CURRENT_USER\Software\JavaSoft\Prefs\com\tim\usong\update]

**Comment by tim4724** [comment]
Ok das ist blöd. Gibt es einen Stacktrace der Fehlermeldung in der Datei: "%Appdata%/uSongServer/log"?

**Comment by Little-Ben** [comment]
Eben nochmal durchgespielt. 
Log vorher gelöscht. 
RegKeys gelöscht (ab tim (inkl)). 
Log-File leider komplett leer.

**Comment by tim4724** [comment]
Du hast nicht zufällig den Internet Explorer deinstalliert oder?
https://support.microsoft.com/de-de/help/4013567/how-to-disable-internet-explorer-on-windows

**Comment by Little-Ben** [comment]
Doch, nutze ihn aber nicht (IE V11). Standardbrowser ist derzeit Cliqz  (basiert auf Firefox)

**Comment by tim4724** [comment]
Verstehe ich das richtig? Der Internet Explorer ist noch da (muss nicht der Standardbrowser sein). :)
Ich werde mal insbesondere schauen warum es kein Log gibt. Dann melde ich mich nochmal.

**Comment by Little-Ben** [comment]
IE V11 deinstalliert, Neustart, keine Veränderung

**Comment by tim4724** [comment]
Nein da hast du mich missverstanden. Der muss installiert sein, sonst geht es nicht :)

Jedes der Fenster ist eigentlich nur eine Webview und der Internet Explorer wird zum Darstellen gebraucht. (Der ist ja normalerweise immer installiert)

**Comment by Little-Ben** [comment]
ups, ok. wieder installiert. 
aber geht dennoch nicht. 
hab mal auch die Firewall (sbsremotesender und java) überprüft: nichts blockiert.

**Comment by tim4724** [comment]
Es ist möglich, dass du eine 32Bit Version der JVM installiert hast. Das ist wohl ein Problem.

Du kannst mal die 32Bit jar ausprobieren, die ich noch hinzugefügt habe: https://github.com/tim4724/u-song-Stage-Monitor-for-Songbeamer/releases/tag/2.2.0

Ich muss mal schauen, wie man das richtig lösen kann. Ein Log und eine Fehlermeldung wäre mindestens gut.

**Comment by Little-Ben** [comment]
jo, das scheints zu sein. geht jetzt.

**Comment by tim4724** [comment]
Bitte beachten: Bei Version 2.2.1 gibt es wieder auch eine 32Bit Variante unter Releases.
https://github.com/tim4724/u-song-Stage-Monitor-for-Songbeamer/releases
